src/listener_address.rs

```diff
@@ -121,6 +125,7 @@ impl Display for ListenerAddress {
             ListenerAddress::FromFdNamed(name) => {
                 write!(f, "sd-listen:{name}")
             }
+            ListenerAddress::Vsock(a) => a.fmt(f),
```

Comment:
Will it be clear that the address is a specifically vsock address? Maybe a prefix like `vsock:` should be used?

Reply:
Good idea, thanks

Reply:
Done as part of `ListenerAddress::Vsock` refactor